content/start.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Getting Started
+description: How to get started with NetObserv operator and CLI
+layout: :theme/page
+---
+
+# Getting Started
+
+There are two independent starting points for using NetObserv:
+
+- By [installing the operator](https://github.com/netobserv/network-observability-operator/blob/main/README.md), allowing H24 network flows monitoring.
```

Comment:
What's "H24"?  24 hours?  I would remove it.

Reply:
yeah, wrong translation from me, it would actually translate better to "24/7 monitoring"

Reply:
/lgtm